blog/2024-04-18-contribution-neurips/index.mdx

```diff
@@ -18,15 +18,14 @@ image: image_1.jpg
 The Swiss Data Science Center presented Renku in the [37th Conference
 on Neural Information Processing Systems (NeurIPS)](https://neurips.cc/Conferences/2023),
 which took place on Dec. 10th to 16th 2023 in New Orleans (US).
-The contribution “Renku: a platform for sustainable data science” presents
+Our contribution [“Renku: a platform for sustainable data science”](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf) presents
 those Renku features that make it a unique platform to support sustainable
 development and use of data and code, and showcases how Renku works in
 different scientific domains.
 
-You can access our [work](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf)
-and [complementary material](https://proceedings.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Supplemental-Datasets_and_Benchmarks.pdf) online.
-
-Please, check our poster and presentation [here](https://neurips.cc/virtual/2023/poster/73464)
+Check out our [paper (https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf),
```

Comment:
I messed up the formatting in my suggestion and link isn't rendering correctly - sorry!
```suggestion
Check out our [paper](https://papers.neurips.cc/paper_files/paper/2023/file/838694e9ab6b0a193b84daaafcac0eed-Paper-Datasets_and_Benchmarks.pdf),
```